_partials/_cloud-connect.md

```diff
@@ -1,14 +1,17 @@
 <Procedure>
 
-### Check your service and connect to it
+## Check your service and connect to it
 
 To ensure a Timescale service is running correctly:
 
-1.  In the [Services section in Timescale Console][services-portal], check that your service is marked as `Running`.
+1. **Check your service**
 
-1. Use PopSQL or psql to connect to your service:
-    - [Setup PopSQL][popsql]: Follow the instructions to easily connect to your service in the UI.
-    - [psql][install-psql]: Connect to your service with the value of `Service URL` from the config file you 
+    In the [Services section in Timescale Console][services-portal], check that your service is marked as `Running`.
+
+1. **Connect to your service**
+    - [Run SQLeditor][run-sqleditor]: in Timescale Console, click `SQLeditor`. 
```

Comment:
What does "Run SQL editor" mean?

Also there should be a space between `SQL` and `editor`

```suggestion
    - [SQL editor][run-sqleditor]: in Timescale Console, click `SQLeditor`. 
```

Reply:
"Run" in this case means "open". We usually use verbs for steps, to be as consistent and as clear as possible. Can be changed to "Open SQL editor" - will this work better?

Reply:
Changed to Open to match the SQL editor page.